eng/dashboardpack/Sdk.in.targets

```diff
@@ -2,15 +2,15 @@
 
   <!-- *** BEGIN *** -->
 
-  <PropertyGroup Condition="'$(NETCoreSdkRuntimeIdentifier)' == '@PlatformRuntime@'">
+  <PropertyGroup>
```

Comment:
Do we still need this to be an `in.targets` file anymore now that we are removing these?